components/x-live-blog-post/src/LiveBlogPost.jsx

```diff
@@ -1,91 +1,80 @@
+				className="x-live-blog-post-controls__back-to-top-link"
 			>
 				Back to top
 			</a>
 		)
 	}
 
-	if (typeof backToTopProp === 'function') {
+	if (typeof backToTop === 'function') {
 		return (
 			<button
-				onClick={backToTopProp}
+				onClick={backToTop}
 				aria-labelledby="Back to top"
-				className={styles['live-blog-post-controls__back-to-top-button']}
+				className="x-live-blog-post-controls__back-to-top-button"
 			>
 				Back to top
 			</button>
 		)
 	}
 }
 
-const LiveBlogPost = (props) => {
-	const {
-		id,
-		postId, // Remove once wordpress is no longer in use
-		title,
-		content, // Remove once wordpress is no longer in use
-		bodyHTML,
-		publishedTimestamp, // Remove once wordpress is no longer in use
-		publishedDate,
-		isBreakingNews, // Remove once wordpress is no longer in use
-		standout = {},
-		articleUrl,
-		showShareButtons = false,
-		byline,
-		ad,
-		backToTop
-	} = props
-
+const LiveBlogPost = ({
+	id,
+	postId, // Remove once wordpress is no longer in use
+	title,
+	content, // Remove once wordpress is no longer in use
+	bodyHTML,
+	publishedTimestamp, // Remove once wordpress is no longer in use
+	publishedDate,
+	isBreakingNews, // Remove once wordpress is no longer in use
+	standout = {},
+	articleUrl,
+	showShareButtons = false,
+	byline,
+	ad,
+	backToTop
+}) => {
 	const showBreakingNewsLabel = standout.breakingNews || isBreakingNews
 
-	const BackToTopComponent = generateBackToTopComponent(backToTop)
-
 	return (
 		<article
-			className={`live-blog-post ${styles['live-blog-post']}`}
+			className="x-live-blog-post"
 			data-trackable="live-post"
 			id={`post-${id || postId}`}
 			data-x-component="live-blog-post"
 		>
-			<div className="live-blog-post__meta">
+			<div className="x-live-blog-post__meta">
 				<Timestamp publishedTimestamp={publishedDate || publishedTimestamp} />
 			</div>
-			{showBreakingNewsLabel && <div className={styles['live-blog-post__breaking-news']}>Breaking news</div>}
-			{title && <h2 className={styles['live-blog-post__title']}>{title}</h2>}
-			{byline && <p className={styles['live-blog-post__byline']}>{byline}</p>}
+			{showBreakingNewsLabel && <div className="x-live-blog-post__breaking-news">Breaking news</div>}
+			{title && <h2 className="x-live-blog-post__title">{title}</h2>}
+			{byline && <p className="x-live-blog-post__byline">{byline}</p>}
 			<div
-				className={`${styles['live-blog-post__body']} n-content-body article--body`}
+				className="x-live-blog-post__body n-content-body article--body"
 				dangerouslySetInnerHTML={{ __html: bodyHTML || content }}
 			/>
-			<div className={styles['live-blog-post__controls']}>
+			<div className="x-live-blog-post__controls">
 				{showShareButtons && <ShareButtons postId={id || postId} articleUrl={articleUrl} title={title} />}
-				{Boolean(BackToTopComponent) && <Fragment>{BackToTopComponent}</Fragment>}
+				<BackToTop backToTop={backToTop} />
```

Comment:
question- do we need to use boolean just in case it shows null in the html?

Reply:
`return null` from a React component won't output anything